go.mod

```diff
@@ -8,13 +8,13 @@ require (
 	github.com/jpillora/ipmath v0.0.0-20180121110145-ebede80a2ab9
 	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
 	github.com/openshift/cloud-credential-operator v0.0.0-20200303191255-cc686af9c1b9
-	github.com/openshift/machine-api-operator v0.2.1-0.20200402110321-4f3602b96da3
+	github.com/openshift/machine-api-operator v0.0.0-20200514040855-461bde5e8c5e
 	github.com/operator-framework/operator-sdk v0.14.0
 	github.com/redhat-cop/operator-utils v0.1.1
 	github.com/scylladb/go-set v1.0.2
 	github.com/spf13/pflag v1.0.5
-	k8s.io/api v0.18.0-rc.1
-	k8s.io/apimachinery v0.18.0-rc.1
+	k8s.io/api v0.18.2
```

Comment:
why this changes? these versions are based on the operator-sdk dependencies and as far as I understand should not be changed.

Reply:
That's a good question, I have to confess that I haven't done that much with golang so far. Whenever I change those numbers back to the old version, v0.18.0-rc.1, `go mod download` changes the version to v0.18.2.

Is it possible that my change of the machine-api-operator to a newer revision (https://github.com/openshift/machine-api-operator/commit/461bde5e8c5e175639187ed78571fdba36afc1ac, had to update this because of https://github.com/openshift/library-go/commit/c1de486c7d47605416673079735c41703b03b7b4, the bitbucket repo is not available anymore) automatically switches k8s.io/api and k8s.io/apimachinery to v0.18.2 because they're using that version, too (see https://github.com/openshift/machine-api-operator/commit/461bde5e8c5e175639187ed78571fdba36afc1ac#diff-37aff102a57d3d7b797f152915a6dc16L30-R33)?